src/components/ProfileSwitcher.js

```diff
@@ -0,0 +1,43 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import Dropdown from './Dropdown';
+
+import { setActiveProfile } from '../store/actions';
+
+import styles from './ProfileSwitcher.module.css';
+
+function ProfileSwitcher() {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const activeProfile = useSelector((state) => state.activeProfile);
+  const profiles = useMemo(
+    () => [
+      {
+        value: 'ch',
+        text: t('profiles.bund'),
```

Comment:
```suggestion
        text: t('profiles.ch'),
```

Let's use the abbrevations as translation keys to avoid german and to have a clear mapping between profile key and translation. Yeah, pretty nitpicking 😉

Reply:
nitpicking is good =)